php/cubex/composer.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "bajb/cubex-min",
+  "license": "MIT",
+  "type": "project",
+  "config": {
+    "optimize-autoloader": true,
+    "preferred-install": "dist",
+    "sort-packages": true
+  },
+  "prefer-stable": true,
+  "require": {
+    "cubex/framework": "^4.0"
```

better use `~` to be sure we are aligned with the doc
```suggestion
    "cubex/framework": "~4.15.0"
```